include/battle.h

```diff
@@ -223,6 +235,8 @@ struct SpecialStatus
     u8 ppNotAffectedByPressure:1;
     u8 faintedHasReplacement:1;
     u8 focusBanded:1;
+    u8 focusSashed:1;
```

Comment:
Looks like potentially a duplicate change with PR #22 ? Same comment applies - I think we need fix the alignment here, maybe slap these both as bitfields of one of the unused fields.

Reply:
correct, had to duplicate some stuff for sturdy and tried to make the change 1:1 with #22 so it is easier to merge.